SimpleCam.xcodeproj/project.xcworkspace/xcshareddata/SimpleCam.xccheckout

```diff
@@ -10,29 +10,29 @@
 	<string>SimpleCam</string>
 	<key>IDESourceControlProjectOriginsDictionary</key>
 	<dict>
-		<key>21835551-A8FF-428F-BD29-5FA8D0C02687</key>
-		<string>https://github.com/LoganWright/SimpleCam.git</string>
+		<key>B08128E43CF8FD47F28F099A420046D02BCA9F92</key>
+		<string>https://github.com/jeffreality/SimpleCam.git</string>
```

Comment:
I don't have a good .gitignore on this project for whatever reason, would it be possible to omit the `.xccheckout` file from the pull?